src/aks-preview/HISTORY.rst

```diff
@@ -11,6 +11,10 @@ To release a new version, please select a new version number (usually plus 1 to
 
 Pending
 +++++++
+
+3.0.0b6
```

Comment:
It seems that 3.0.0b6 has been released, please use a new version.
https://github.com/Azure/azure-cli-extensions/blob/c9e19421a81ff560878997156cfc21a40c8be0a2/src/aks-preview/setup.py#L12